packages/create-react-app/createReactApp.js

```diff
@@ -182,27 +181,31 @@ createApp(
 );
 
 function createApp(name, verbose, version, useNpm, template) {
-  const root = path.resolve(name);
-  const appName = path.basename(root);
+  const appName = `portal-${name}-frontend`;
+  const root = path.resolve(appName);
+  const microappId = path.basename(name);
 
   checkAppName(appName);
-  fs.ensureDirSync(name);
-  if (!isSafeToCreateProjectIn(root, name)) {
+  fs.ensureDirSync(appName);
+  if (!isSafeToCreateProjectIn(root, appName)) {
     process.exit(1);
   }
 
   console.log(`Creating a new React app in ${chalk.green(root)}.`);
```

Comment:
add ``@infosight/${appName}` to the console.log

Reply:
Added